presage/src/manager/confirmation.rs

```diff
@@ -87,7 +88,50 @@ impl<S: Store> Manager<S, Confirmation> {
 
         let profile_key = ProfileKey::generate(profile_key);
 
+        let aci_identity_key_pair = IdentityKeyPair::generate(&mut rng);
+        let pni_identity_key_pair = IdentityKeyPair::generate(&mut rng);
+
+        let (_aci_pre_keys, aci_signed_pre_key, _aci_pq_pre_keys, aci_pq_last_resort_pre_key) =
+            libsignal_service::pre_keys::replenish_pre_keys(
+                &mut self.store,
+                &aci_identity_key_pair,
+                &mut rng,
+                true,
+                0,
+                0,
+            )
+            .await?;
```

Comment:
You are aware this block is in AccountManager::register_account right?

Reply:
I meant all the way up to the activation request, but scrolling took ages.

Reply:
Yes, this is the part that's currently broken anyways. I was gonna work on making sure we have a functional place for that, I don't know why it's duplicated anymore.

Reply:
Removed from `presage` 🥳